Swiftfin/Views/SettingsView/SettingsView.swift

```diff
@@ -185,6 +187,22 @@ struct SettingsView: View {
                         Text(appearance.localizedName).tag(appearance.rawValue)
                     }
                 }
+
+                Button {
+                    settingsRouter.route(to: \.fontPicker) { font in
```

Comment:
We still don't need to pass an `onFontPick` closure around and I don't think popping on a selection is a nice behavior while a user is looking around.

Reply:
hmm okay..